.github/actions/restore-cache-azcopy/action.yml

```diff
@@ -45,9 +45,9 @@ runs:
           fi            
         fi
     env:
-      AZURE_AKS_CACHE_STORAGE_ACCOUNT: f723719aa87a34622b5f7f3
-      AZURE_AKS_CACHE_SHARE_NAME: pvc-f6a4089f-b082-4bee-a3f9-c3e1c0c02d8f
-      AZURE_AKS_WIN_CACHE_SHARE_NAME: pvc-71dec4f2-0d44-4fd1-a2c3-add049d70bdf
+      AZURE_AKS_CACHE_STORAGE_ACCOUNT: 0lhymkr4ctmi
```

Comment:
Won't block this PR on it, but given this account name is random / cluster dependent and we could be in a world where we either have multiple clusters or spin up new clusters ad-hoc this account name should be persisted as part of the sas-token file and read out of there IMO